lib/subprocess.js

```diff
@@ -74,7 +74,7 @@ class SubProcess extends EventEmitter {
     }
 
     // return a promise so we can wrap the async behavior
-    return new B((resolve, reject) => {
+    return await new B((resolve, reject) => {
```

Comment:
I'm not quite sure if the resulting promise should be awaited, since it is anyway going to be wrapped into a promise, because the method is marked as async.

Reply:
This was the style that the team agreed upon when moving to ES2015 and async/await.